ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -36,10 +36,6 @@ vacuum_cost_delay = 20ms
 
 # WRITE AHEAD LOG
 
-archive_mode = on
-archive_command = 'test ! -f {{ postgresql_archive_dir }}/%f && cp %p {{ postgresql_archive_dir }}/%f'
-archive_timeout = 60
-
```

Comment:
@javierwilson isn't this is going to interfere with pgbackrest? https://github.com/dimagi/commcarehq-ansible/pull/917/files#diff-1de0fd6b211ae1ef3b9020f83c0635d0R32